app/models/manageiq/providers/vmware/infra_manager/inventory/collector.rb

```diff
@@ -2,14 +2,21 @@ class ManageIQ::Providers::Vmware::InfraManager::Inventory::Collector
   include PropertyCollector
   include Vmdb::Logging
 
-  def initialize(ems, threaded: true)
```

Comment:
@Fryguy Okay I dropped the run_once and threaded, replaced it with a `Collector#refresh` method and the save_inventory thread was the only place that I had to do RAILS_ENV != test